lib/iis_shc_issuer.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+require 'iis_shc_issuer/qbp_client'
+require 'iis_shc_issuer/errors'
+
+module IISSHCIssuer
+  class Error < StandardError; end
```

Comment:
Is this actually used for anything?

Reply:
I removed it since it wasn't being used anywhere. We can always add it back if we need it in the future.